pkgs/applications/networking/cluster/eks-node-viewer/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "eks-node-viewer";
+  version = "2022-12-10";
+
+  src = fetchFromGitHub {
+    owner = "awslabs";
+    repo = pname;
+    rev = version;
```

Comment:
The repo has no tags or branch named like this.

Reply:
Yes, they still are not very mature, but I understood that timestamp version just fetches a commit for that date. Just tried, it worked.

Reply:
For ofborg this is not working https://logs.nix.ci/?key=nixos/nixpkgs.210510&attempt_id=35fd350a-60b9-4a4c-a227-c8f2a5c45d17

Reply:
Ah, got it, let me watch the repo if they add a tag for some initial release. Or nudge them.